src/webapp/components/progress/Progress.tsx

```diff
@@ -0,0 +1,55 @@
+import Typography from "@material-ui/core/Typography";
+import styled, { css } from "styled-components";
+import React from "react";
+import customTheme from "$/webapp/pages/app/themes/customTheme";
+import { Step } from "$/webapp/pages/dashboard/mock";
+
+type Props = {
+    value: Step[];
+};
+
+export type ListElementProps = {
+    $variant: Step["status"];
+};
+
+export const Progress: React.FC<Props> = React.memo(({ value }) => {
+    return (
+        <Container>
+            {value.map(step => (
+                <ListElement $variant={step.status} key={step.position}>
```

Comment:
Having a "$" for this case seems odd, normally we use a symbol in front of a variable to express something about it, like if you have a dom ref:

```js
const $domEl = document.querySelector("#my_div");
```

but in this case the value of the prop looks like a "normal" one so can we keep this as "variant"? Not sure if `styled` will complain about that.

Reply:
Oh, that's because at one point I changed the name from 'status' to 'variant.' Using '$' allows us to repeat the name for styling variants, which is completely safe with styled-components. I'm changing the naming back to 'status,' but if you prefer a different approach, please let me know.